libafl/src/mutators/mod.rs

```diff
@@ -1,5 +1,8 @@
-//! [`Mutator`]`s` mutate input during fuzzing.
-
+//! [`Mutator`]`s` mutate input during fuzzing. These can be used standalone or in combination with other mutators to explore the input space more effectively.
+//!
+//! The following mutators are provided by this module:
+//! - [`MOpt`]: The mutation scheduler used in AFL++. It uses a modified Particle Swarm Optimization algorithm to determine an optimal distribution of mutators.
+//! - [`GramatronRandomMutator`]: A random mutator using grammar automatons to perform grammar-aware fuzzing.
```

Comment:
Is your plan to add all of the mutators here? I wonder if it wouldn't be better to document each module? I fear if we do it here it'll be out of date soon...

Reply:
https://docs.rs/libafl/latest/libafl/mutators/index.html#modules

Like, maybe we should improve this?

Reply:
Oh yeah i was wondering which would be better. To document the module or the submodules but as the documentation lists the submodules I'll move it there